build/ferretdb/production.Dockerfile

```diff
@@ -88,6 +88,7 @@ go version -m bin/ferretdb
 bin/ferretdb --version
 EOF
 
+RUN mkdir /state
 
 # stage for binary only
```

Comment:
@AlekSi why is this stage needed at all? I see it's not used anywhere later.

Reply:
See `TARGET: production-binary` in the Taskfile